src/script/script.h

```diff
@@ -69,143 +69,143 @@ std::vector<unsigned char> ToByteVector(const T& in)
-    OP_5 = 0x55,
-    OP_6 = 0x56,
-    OP_7 = 0x57,
-    OP_8 = 0x58,
-    OP_9 = 0x59,
-    OP_10 = 0x5a,
-    OP_11 = 0x5b,
-    OP_12 = 0x5c,
-    OP_13 = 0x5d,
-    OP_14 = 0x5e,
-    OP_15 = 0x5f,
-    OP_16 = 0x60,
+    OP_0 = 0x00, // push "" onto the stack (which is an empty array of bytes)
+    OP_FALSE = OP_0, // push "" onto the stack (which is an empty array of bytes)
+    OP_PUSHDATA1 = 0x4c, // read the next byte as N and push the next N bytes as an array onto the stack
+    OP_PUSHDATA2 = 0x4d, // read the next 2 bytes as N and push the next N bytes as an array onto the stack
+    OP_PUSHDATA4 = 0x4e, // // read the next 4 bytes as N and push the next N bytes as an array onto the stack
+    OP_1NEGATE = 0x4f, // push "\x81" onto the stack (which is interpreted as -1 by numerical opcodes)
+    OP_RESERVED = 0x50, // mark tx invalid unless occurring in an unexecuted OP_IF branch
+    OP_1 = 0x51, // push "\x01" onto the stack (which is interpreted as 1 by numerical opcodes)
+    OP_TRUE=OP_1, // push "\x01" onto the stack (which is interpreted as 1 by numerical opcodes)
+    OP_2 = 0x52, // push "\x02" onto the stack (which is interpreted as 2 by numerical opcodes)
+    OP_3 = 0x53, // push "\x03" onto the stack (which is interpreted as 3 by numerical opcodes)
+    OP_4 = 0x54, // push "\x04" onto the stack (which is interpreted as 4 by numerical opcodes)
+    OP_5 = 0x55, // push "\x05" onto the stack (which is interpreted as 5 by numerical opcodes)
+    OP_6 = 0x56, // push "\x06" onto the stack (which is interpreted as 6 by numerical opcodes)
+    OP_7 = 0x57, // push "\x07" onto the stack (which is interpreted as 7 by numerical opcodes)
+    OP_8 = 0x58, // push "\x08" onto the stack (which is interpreted as 8 by numerical opcodes)
+    OP_9 = 0x59, // push "\x09" onto the stack (which is interpreted as 9 by numerical opcodes)
+    OP_10 = 0x5a, // push "\x0A" onto the stack (which is interpreted as 10 by numerical opcodes)
+    OP_11 = 0x5b, // push "\x0B" onto the stack (which is interpreted as 11 by numerical opcodes)
+    OP_12 = 0x5c, // push "\x0C" onto the stack (which is interpreted as 12 by numerical opcodes)
+    OP_13 = 0x5d, // push "\x0D" onto the stack (which is interpreted as 13 by numerical opcodes)
+    OP_14 = 0x5e, // push "\x0E" onto the stack (which is interpreted as 14 by numerical opcodes)
+    OP_15 = 0x5f, // push "\x0F" onto the stack (which is interpreted as 15 by numerical opcodes)
+    OP_16 = 0x60, // push "\x10" onto the stack (which is interpreted as 16 by numerical opcodes)
 
     // control
-    OP_NOP = 0x61,
-    OP_VER = 0x62,
-    OP_IF = 0x63,
-    OP_NOTIF = 0x64,
-    OP_VERIF = 0x65,
-    OP_VERNOTIF = 0x66,
-    OP_ELSE = 0x67,
-    OP_ENDIF = 0x68,
-    OP_VERIFY = 0x69,
-    OP_RETURN = 0x6a,
+    OP_NOP = 0x61, // do nothing
+    OP_VER = 0x62, // mark tx invalid unless occuring in an unexecuted OP_IF branch
+    OP_IF = 0x63, // if top stack value true, execute the statement
```

Comment:
We should probably be consistent in describing how Script deals with true/false, since it's later described as `if top stack value != 0`.  Maybe add a section along the lines of:

``` 
/** 
* Opcodes that take a true/false value will evaluate the following as false:
*     an empty vector
*     a vector (of any length) of all zero bytes 
*     a single byte of "\x80" ('negative zero')
*     a vector (of any length) of all zero bytes except the last byte is "\x80"
*
* Any other value will evaluate to true.
* /
```

I'd also say ` if the top stack value is true,`, and indicate that it consumes the stack item.